hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/datanodes/datanodes.tsx

```diff
@@ -436,8 +453,39 @@ export class Datanodes extends React.Component<Record<string, object>, IDatanode
     console.log(current, pageSize);
   };
 
+  onSelectChange = (newSelectedRowKeys:any) => {
+    this.setState({
+      selectedRowKeys: newSelectedRowKeys
+    });
+  };
+
+  onDisable = (record:any) => {
+    // Will return Disable Checkbox records for remove whos Record state is not DEAD and Opeartional State=['IN_SERVICE','ENTERING_MAINTENANCE','DECOMMISSIONING']
+    // Enable Checkbox for remove who's State is Dead or Operation State=['Decommissioned','In_Maintenance']
+    if (record.state !== 'DEAD') {
+      return record.opState === 'IN_SERVICE' || record.opState === 'ENTERING_MAINTENANCE' || record.opState === 'DECOMMISSIONING';
```

Comment:
Shouldn't this if check also have "`IN_MAINTENANCE`"  and "`DECOMMISSIONED`" ?

Reply:
in latest commit it was corrected